branching_xblock/static/js/src/studio_editor.js

```diff
@@ -12,6 +12,17 @@ function BranchingStudioEditor(runtime, element, data) {
   const $saveBtn    = $root.find('.save-button');
   const $cancelBtn  = $root.find('.cancel-button');
 
+  const $actionsBar = $root.next('.xblock-actions');
+  if ($actionsBar.length && !$actionsBar.data('branching-sticky')) {
+    $actionsBar.css({
+      position: 'sticky',
+      bottom: 0,
+      'z-index': 10,
+      'box-shadow': '0 -2px 6px rgba(0, 0, 0, 0.05)'
+    });
+    $actionsBar.data('branching-sticky', true);
+  }
+
```

Comment:
The sticky bottom buttons things works with just the CSS changes from the studio_editor.css. I was able to verify that this block of code is not needed, by commenting it out on the `master` devstack. Is this being required in Sumac or Teak? If not, we can remove it. If needed, we should add a comment here mentioning the reason and make a not about removing it in the future.

<img width="1502" height="1218" alt="image" src="https://github.com/user-attachments/assets/36395989-0d26-4401-8aad-05b497876142" />